src/protocol/handshake.rs

```diff
@@ -112,7 +112,7 @@ impl Handshake for InnerNode {
                 request.extend_from_slice(b"Connect-As: Peer\r\n");
                 request.extend_from_slice(format!("Public-Key: {}\r\n", base58_pk).as_bytes());
                 request.extend_from_slice(format!("Session-Signature: {}\r\n", sig).as_bytes());
-                request.extend_from_slice(b"X-Protocol-Ctl: txrr=1\r\n");
+                request.extend_from_slice(b"X-Protocol-Ctl: txrr=1;ledgerreplay=1\r\n");
```

Comment:
what's the rationale behind this? we should probably document it

Reply:
Added a short comment here and a longer explanation in the SPEC file.